### ISSUE TITLE
Wrongfully accepting interface definition in contract body

### ISSUE THREAD

**Issue by nikita-fuchs** [aeternity/aesophia_http#63]
The compiler doesn't complain if you put a contract function interface somewhere into the contract, like https://studio.aepps.com?highlight=19-5-17-1&contract=036202a2-5 (corresponding tab doesn't focus automatically yet, you need to click it) 
When you try to deploy it, it of course throws you an error.

**Comment by dincho** [comment]
Isn't this a feature ? I found it in the tests https://github.com/aeternity/aesophia/blob/lima/test/contracts/address_literals.aes ?

**Comment by radrow** [comment]
No, because these entrypoints do have implementation. The case is when you provide just an entrypoint declaration without the body

**Comment by radrow** [comment]
Hey, wrong repo. This is an issue with `aesophia`, not the http interface
